beetsplug/subsonicupdate.py

```diff
@@ -40,60 +39,56 @@ def __init__(self):
         super(SubsonicUpdate, self).__init__()
 
         # Set default configuration values
-        config['subsonic'].add({
+        self.config.add({
             'user': 'admin',
-            'pass': 'admin',
+            'password': 'admin',
             'url': 'http://localhost:4040',
         })
 
-        config['subsonic']['pass'].redact = True
+        self.config['password'].redact = True
         self.register_listener('import', self.start_scan)
 
     @staticmethod
-    def __create_token():
+    def __create_token(self):
```

Comment:
[Static methods](https://docs.python.org/3/library/functions.html#staticmethod) do not get a `self` parameter.

Reply:
Ah! Makes sense. Thanks for link.